descriptors_noncommercial.inc

```diff
@@ -3018,8 +3020,17 @@
                            !normalisation factor
                            descriptor_out%x(i_desc)%grad_covariance_cutoff(:,:) = descriptor_out%x(i_desc)%grad_covariance_cutoff(:,:) / n_products
                         end if cutoff_type_grad
-
+                        
                      end if calc_grad_descriptor
+                     
```

Comment:
Can you please remove this or change it to use our built in `print()` interface.

Reply:
Yes, that's a mistake, I'll remove all those prints.